tests/integ/test_udf.py

```diff
@@ -1557,54 +1566,76 @@ def add_one_df_pandas_annotation(df: pandas.DataFrame) -> pandas.Series:
     )
 
 
-@pytest.mark.skipif(not is_pandas_and_numpy_available, reason="pandas is required")
+@pytest.mark.skipif(not is_pandas_available, reason="pandas is required")
 @pytest.mark.parametrize(
     "_type, data, expected_types, expected_dtypes",
     [
         (
             IntegerType,
             [[4096]],
-            (numpy.int16, int, numpy.short),
+            ("<class 'numpy.int16'>", "<class 'int'>"),
```

Comment:
Are these changes expected?

Reply:
Yes these are expected since `parameterize` is evaluated before `skip` is evaluated. So the test will try to evaluate `numpy.int16` which fails when `pandas` is not installed. So I converted these into string comparison as a workaround.